cosmos/testing/e2e/precompile/staking/staking_test.go

```diff
@@ -79,7 +79,7 @@ var _ = Describe("Staking", func() {
 	})
 
 	It("should call functions on the precompile directly", func() {
```

Comment:
These tests need to be revised after the e2e localnet node config is updated to have multiple validators